validator/node/node.go

```diff
@@ -60,7 +62,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 	"google.golang.org/protobuf/encoding/protojson"
-	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/util/yaml"
```

Comment:
any reason to change this to a different package ?

Reply:
the k8s package will unmarshal yamls despsite only having `json:` in the struct, in our case the proto. since we manually wrote the json object that parses before I added `yaml:` settings but since i'm using the proto now it doesn't auto generate with it. the k8s library is more flexible in this case